crates/mock-download-server/src/lib.rs

```diff
@@ -33,10 +34,10 @@ pub fn new() -> Builder {
         data: Data {
             tokens: HashMap::new(),
             keys: Vec::new(),
-            revoked_signatures: SignedPayload::new(&RevocationInfo {
-                revoked_content_sha256: Vec::new(),
-                expires_at: OffsetDateTime::now_utc() + Duration::days(99),
-            })
+            revoked_signatures: SignedPayload::new(&RevocationInfo::new(
+                Vec::new(),
+                datetime!(2025-01-01 0:00 UTC),
```

Comment:
```suggestion
                datetime!(3025-01-01 0:00 UTC),
```

This is test code, so having a date like this is ok, but let's avoid having to trip over it in just a few months?